src/main/drivers/bus_spi.h

```diff
@@ -59,7 +59,7 @@ typedef enum {
 #elif defined(STM32F7)
     SPI_CLOCK_SLOW          = 256, //00.42188 MHz
     SPI_CLOCK_STANDARD      = 16,  //06.57500 MHz
-    SPI_CLOCK_FAST          = 8,   //13.50000 MHz
+    SPI_CLOCK_FAST          = 4,   //27.00000 MHz
```

Comment:
Maybe use `SPI_CLOCK_13_5_MHZ`, `SPI_CLOCK_27_MHZ`, `SPI_CLOCK_54_MHZ`?

Reply:
The problem here is that each processor has a different max. speed from which other speeds are a power of two division.

| Proc | Speed (MHz) |
|------|-------------|
| F4 | 42 |
| F7 | 54 |
| H7 | 50 |
| G4 | 42.5 |

So we can't be too exact, hence these approx. definitions.

Reply:
Not sure why this is so complicated... Then just add `SPI_CLOCK_FLASH_SLOW`, `_STANDARD` and `_FAST` enumerations which are appropriate for the range of devices and use these exclusively for the flash devices rather than changing a general enumeration used for other devices.

Reply:
Or better yet, add a `spiCalculateDivider()` function that takes the desired MHz and returns the closest divider (without going over) for the current MCU architecture. The set the desired speed by the official datasheet specs.

Reply:
I agree this is good idea. I'll do that.

Reply:
Done. I've also added both read and normal SPI max clock frequencies for each FLASH memory type. Logging can run at max speed on most devices except when being read which isn't the operation that affects logging.